docs.json

```
@@ -303,6 +303,23 @@
               }
             ]
           },
+          {
+            "group": "Edge Functions",
+            "icon": "/images/icons/edge-function-icon.svg",
+            "pages": [
+              {
+                "group": "Utility Functions",
+                "pages": [
+                  "components/edge-functions/maintenance-page",
+                  "components/edge-functions/mailchimp",
+                  "components/edge-functions/pdforge",
+                  "components/edge-functions/slack-message",
+                  "components/edge-functions/stripe-billing-portal",
+                  "components/edge-functions/sendgrid"
```

let's sort alphabetically :) 

```suggestion
                  "components/edge-functions/maintenance-page",
                  "components/edge-functions/mailchimp",
                  "components/edge-functions/pdforge",
                  "components/edge-functions/sendgrid",
                  "components/edge-functions/slack-message",
                  "components/edge-functions/stripe-billing-portal"
```